examples/plugins/cli-cmd/hello/hello_option.py

```diff
@@ -0,0 +1,20 @@
+from avocado.core.output import LOG_UI
```

Comment:
These "loose" plugin files can not be easily used by a user, as per `examples/plugins/README.rst`.  Even though it's more work, I think it'd make sense to let users easily activate and run those examples.  Also, we can think about even running them to test the examples don't bit rot.

Reply:
Hum.. I was just trying to hide code from the .rst files. Not sure if for the "didactic perspective" this "extra work" is worthed.

At this stage, users (contributors) are learning how to "write plugins" and not how to "use plugins". I do have a mix of feelings here.

Reply:
Well, my opinion is, two new directories, two new (almost identical) `setup.py` files, and we don't break what we document.  For instance, I *believe* these plugins will crash on `avocado plugins` due to a lack of `description =`.  That's part of the value for having them easy to deploy.